cg/cli/upload/base.py

```diff
@@ -77,12 +77,12 @@ def upload(context: click.Context, case_id: str | None, restart: bool):
 
         if Workflow.BALSAMIC in case.data_analysis:
             upload_api = BalsamicUploadAPI(config_object)
-        elif case.data_analysis == Workflow.RNAFUSION:
-            upload_api = RnafusionUploadAPI(config_object)
         elif case.data_analysis == Workflow.MIP_RNA:
             upload_api = MipRNAUploadAPI(config_object)
         elif case.data_analysis == Workflow.MICROSALT:
             upload_api = MicrosaltUploadAPI(config_object)
+        elif case.data_analysis in {Workflow.RNAFUSION, Workflow.TOMTE}:
+            upload_api = NfAnalysisUploadAPI(config_object)
```

Comment:
heads up for merge conflicts @sofstam in #3108. Maybe this class could be used for taxprofiler uploads too

Reply:
woops! thanks for linking!

Reply:
Sounds good, thanks for the heads-up @diitaz93